package.json

```diff
@@ -62,16 +69,32 @@
     "@fortawesome/free-solid-svg-icons": "^5.12.1",
     "@fortawesome/react-fontawesome": "^0.1.9",
     "bestzip": "^2.1.5",
+    "codemirror": "^5.53.2",
```

Comment:
I'm going to have to move all these back to dev dependencies so electron users don't have to fetch all these dependencies post-build.